tools/terraform-bundle/README.md

```diff
@@ -80,10 +87,10 @@ The `terraform` block defines which version of Terraform will be included
 in the bundle. An exact version is required here.
 
 The `providers` block defines zero or more providers to include in the bundle
-along with core Terraform. Each attribute in this block is a provider name,
-and its value is a list of version constraints. For each given constraint,
-`terraform-bundle` will find the newest available version matching the
-constraint and include it in the bundle.
+along with core Terraform. Each attribute is a provider name, and its value is a
+blick with the list of version constraints and (optional) source. For each given
```

Comment:
```suggestion
block with the list of version constraints and (optional) source. For each given
```

Reply:
GitHub says this is resolved but it still seems to say "blick" in the version I'm reviewing here. :thinking: